gradio/routes.py

```diff
@@ -274,11 +274,26 @@ async def proxy_to_node(
             request.method, httpx.URL(url), headers=headers
         )
         node_response = await App.client.send(new_request)
+        content = node_response.content
+        response_headers = dict(node_response.headers)
+        response_headers.pop("content-length", None)
+        user_agent = request.headers.get("user-agent", "").lower()
+        is_safari = "safari" in user_agent and "chrome" not in user_agent and "chromium" not in user_agent
+        if is_safari:
+            response_headers = {
+                "Access-Control-Allow-Origin": "*",
+                "Cross-Origin-Opener-Policy": "same-origin",
+                "Cross-Origin-Embedder-Policy": "require-corp",
+            }
+        if request.url.path.endswith('.js'):
+            response_headers["Content-Type"] = "application/javascript; charset=utf-8"
+        elif request.url.path.endswith('.css'):
+            response_headers["Content-Type"] = "text/css; charset=utf-8"
```

Comment:
Again, do we want to do this for every browser or just Safari? If we don't need to do it for non-Safari browsers, it would be preferable to only make these changes for Safari for performance reasons